src/tests/dictionaryConcepts/container/CreateConcept.test.jsx

```diff
@@ -157,6 +157,8 @@ describe('Test suite for dictionary concepts components', () => {
   it('should handle form completion and submission with correct data', () => {
     const conceptName = { target: { name: 'name', value: 'test concept' } };
     wrapper.find('#concept-name').simulate('change', conceptName);
+    const conceptClass = { target: { name: 'concept_class', value: 'Procedure' } };
```

Comment:
Is this change part of the ticket?

Reply:
I added this test to increase the coverage. There was no file related to my ticket that needed to be tested